### ISSUE TITLE
Voicing adds listeners that block MagnifiedNode's drag listeners

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#205]
While working on https://github.com/phetsims/friction/issues/203, I ran into an issue where when I mixin Voicing to MagniedNode, it blocks the input going to the children that have the FrictionDragListener added to them. Voicing adds a couple of of input listeners (some via MouseHighlighting.js), and it seems that since that is the parent Node, it blocks input to the nodes below. I didn't have time to experiment with setting different values of pickable. But would that work?

Here is the test case, apply this patch, and note that friction's magnified view doesn't drag anymore.

```diff

Index: js/friction/view/magnifier/MagnifierNode.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/friction/view/magnifier/MagnifierNode.js b/js/friction/view/magnifier/MagnifierNode.js
--- a/js/friction/view/magnifier/MagnifierNode.js	(revision b9d556f66e34b2328d0eb3f1172ef43d20a28206)
+++ b/js/friction/view/magnifier/MagnifierNode.js	(date 1626715933464)
@@ -13,6 +13,7 @@
 import Vector2 from '../../../../../dot/js/Vector2.js';
 import Shape from '../../../../../kite/js/Shape.js';
 import FocusHighlightPath from '../../../../../scenery/js/accessibility/FocusHighlightPath.js';
+import Voicing from '../../../../../scenery/js/accessibility/voicing/Voicing.js';
 import Circle from '../../../../../scenery/js/nodes/Circle.js';
 import HBox from '../../../../../scenery/js/nodes/HBox.js';
 import Node from '../../../../../scenery/js/nodes/Node.js';
@@ -62,6 +63,8 @@
     options.tandem = tandem;
     super( options );
 
+    this.initializeVoicing( options );
+
     // add container for clipping
     this.container = new Node();
     this.addChild( this.container );
@@ -371,4 +374,6 @@
   return Shape.bounds( dragArea.bounds.withOffsets( 0, 40, 0, 0 ) );
 }
 
+Voicing.compose( MagnifierNode );
+
 export default MagnifierNode;
\ No newline at end of file

```

@jonathanolson, note that this is a general problem for any Node mixing in Voicing that may have children with listeners. What do you recommend? @jessegreenberg, will you include a link to the similar issues that you found while we discussed this?

**Comment by jonathanolson** [comment]
I'm not clear on why this would be "blocking" input, could we pair on this sometime?

**Comment by zepumph** [comment]
So after a bit more investigation, it looks like I found and fixed the problem.

Here is the pertinent Node structure (in relative child order)

```
MagnifierNode
  topBookBackground
      dragArea <- has DragListener
      background <- has DragListener
  surrounding black stroked rectangle

```

 When I mixed in Voicing to MagnifierNode, note that we added input listeners to MagnifierNode. Once we did that, the surrounding black stroked rectangle was absorbing the hit, and thus the dragListener Nodes were obscured by this. Before, pickable: null had passthrough behavior on the surrounding black stroked rectangle, but one MagnifierNode got an inputListener, it was picked up by the hitTest. Setting it to `pickable:false` fixed this.

@jonathanolson will you review the commit and explanation and confirm that this makes sense?
